[2022-05-17 01:30:53,570] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [queued]>
[2022-05-17 01:30:53,582] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [queued]>
[2022-05-17 01:30:53,583] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 01:30:53,584] {taskinstance.py:1242} INFO - Starting attempt 4 of 7
[2022-05-17 01:30:53,585] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 01:30:53,598] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_run.model.jaffle_shop.int_orders> on 2022-05-15 16:00:47.250021+00:00
[2022-05-17 01:30:53,602] {standard_task_runner.py:52} INFO - Started process 104 to run task
[2022-05-17 01:30:53,606] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_dag', 'dbt_run.model.jaffle_shop.int_orders', 'scheduled__2022-05-15T16:00:47.250021+00:00', '--job-id', '534', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_dag.py', '--cfg-path', '/tmp/tmp3k7gm3je', '--error-file', '/tmp/tmpm1m6bnuw']
[2022-05-17 01:30:53,608] {standard_task_runner.py:77} INFO - Job 534: Subtask dbt_run.model.jaffle_shop.int_orders
[2022-05-17 01:30:53,657] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-15T16:00:47.250021+00:00 [running]> on host 9d5de636d688
[2022-05-17 01:30:53,716] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=luciano.naveiro
AIRFLOW_CTX_DAG_ID=model_level_dbt_dag
AIRFLOW_CTX_TASK_ID=dbt_run.model.jaffle_shop.int_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-05-15T16:00:47.250021+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-15T16:00:47.250021+00:00
[2022-05-17 01:30:53,718] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-17 01:30:53,719] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt --no-write-json run --target dev --models int_orders --profiles-dir /opt/dbt/jaffle_shop --project-dir /opt/dbt/jaffle_shop']
[2022-05-17 01:30:53,731] {subprocess.py:85} INFO - Output:
[2022-05-17 01:30:56,810] {subprocess.py:89} INFO - 01:30:56  Running with dbt=1.1.0
[2022-05-17 01:30:56,940] {subprocess.py:89} INFO - 01:30:56  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-17 01:30:56,949] {subprocess.py:89} INFO - 01:30:56
[2022-05-17 01:30:57,118] {subprocess.py:89} INFO - 01:30:57  Concurrency: 1 threads (target='dev')
[2022-05-17 01:30:57,120] {subprocess.py:89} INFO - 01:30:57
[2022-05-17 01:30:57,132] {subprocess.py:89} INFO - 01:30:57  1 of 1 START view model dev.int_orders ......................................... [RUN]
[2022-05-17 01:30:57,256] {subprocess.py:89} INFO - 01:30:57  1 of 1 OK created view model dev.int_orders .................................... [[32mCREATE VIEW[0m in 0.12s]
[2022-05-17 01:30:57,363] {subprocess.py:89} INFO - 01:30:57
[2022-05-17 01:30:57,365] {subprocess.py:89} INFO - 01:30:57  Finished running 1 view model in 0.41s.
[2022-05-17 01:30:57,370] {subprocess.py:89} INFO - 01:30:57
[2022-05-17 01:30:57,372] {subprocess.py:89} INFO - 01:30:57  [32mCompleted successfully[0m
[2022-05-17 01:30:57,374] {subprocess.py:89} INFO - 01:30:57
[2022-05-17 01:30:57,376] {subprocess.py:89} INFO - 01:30:57  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-05-17 01:30:58,402] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-17 01:30:58,432] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_dag, task_id=dbt_run.model.jaffle_shop.int_orders, execution_date=20220515T160047, start_date=20220517T013053, end_date=20220517T013058
[2022-05-17 01:30:58,471] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-17 01:30:58,524] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
